packages/template-retail-react-app/app/pages/checkout/partials/shipping-multi-address.jsx

```diff
@@ -0,0 +1,540 @@
+    return (
+        <List spacing={1.5} flex={1} aria-label="Product attributes">
+            {variationAttributes &&
+                variationAttributes.length > 0 &&
+                variationAttributes.map((attr) => {
+                    const value = attr.values?.find((v) => v.value === variationValues[attr.id])
+                    return (
+                        <ListItem key={attr.id}>
+                            <Text lineHeight={1} color="gray.700" fontSize="sm">
+                                {attr.name || attr.id}: {value?.name || value?.value || ''}
+                            </Text>
+                        </ListItem>
+                    )
+                })}
+            {includeQuantity && (
+                <ListItem>
+                    <Text lineHeight={1} color="gray.700" fontSize="sm">
+                        {formatMessage({
+                            id: 'shipping_multi_address.quantity.label',
+                            defaultMessage: 'Quantity'
+                        })}
+                        : {variant.quantity}
+                    </Text>
+                </ListItem>
+            )}
+        </List>
+    )
+}
+
+MultiShippingItemAttributes.propTypes = {
+    variant: PropTypes.object.isRequired,
+    includeQuantity: PropTypes.bool
+}
+
+const ShippingMultiAddress = ({
+    basket,
+    onSubmit,
+    submitButtonLabel,
+    addNewAddressLabel,
+    noItemsInBasketMessage,
+    deliveryAddressLabel
+}) => {
+    const {formatMessage} = useIntl()
+    const {currency} = useCurrency()
+    const productIds = basket?.productItems?.map((item) => item.productId).join(',')
+    const {
+        data: productsMap,
+        isLoading: productsLoading,
+        error: productsError
+    } = useProducts(
+        {parameters: {ids: productIds, allImages: true}},
+        {
+            enabled: Boolean(productIds),
+            select: (data) => {
+                return (
+                    data?.data?.reduce((acc, p) => {
+                        acc[p.id] = p
+                        return acc
+                    }, {}) || {}
+                )
+            }
+        }
+    )
+    const {data: customer} = useCurrentCustomer()
+    const addresses = customer?.addresses || []
+    const [selectedAddresses, setSelectedAddresses] = useState({})
+
+    // Early return after hooks
+    if (!basket?.productItems?.length) {
+        return (
+            <Box
+                p={8}
+                textAlign="center"
+                color="gray.500"
+                role="status"
+                aria-live="polite"
+                aria-label={formatMessage(noItemsInBasketMessage)}
+            >
+                {formatMessage(noItemsInBasketMessage)}
+            </Box>
+        )
+    }
+
+    // Loading state
+    if (productsLoading) {
+        return (
+            <Box
+                p={8}
+                textAlign="center"
+                color="gray.500"
+                role="status"
```

Comment:
why do we need this role here? I'd suggest we remove it along with the aria-*. Same as the error rendering block below?